homeassistant/components/roomba/irobot_base.py

```diff
@@ -219,6 +219,15 @@ def extra_state_attributes(self):
 
         return state_attrs
 
+    def get_cleaning_time(self, state) -> int:
```

Comment:
Why not pass `mission_state` here. We already get it before calling this function.